src/http/validations/device.js

```diff
@@ -34,4 +34,19 @@ const updateDevice = (req, res, next) => {
   return next();
 };
 
-export default { getSingleDevice, updateDevice };
+const verifyDeviceLabel = async (req, res, next) => {
+  const { label } = req.body;
+  const { user, userType } = res.locals;
+  const deviceMatch = {
+    label: label.toLowercase(),
```

Comment:
The label field here is optional, you should check if a value is provided for the field before converting it to lowercase, I think this is what is making your CI build to fail.

Reply:
Thanks for pointing it out. Fixed